pkg/testcase/selinux.go

```diff
@@ -0,0 +1,106 @@
+package testcase
+
+import (
+    "fmt"
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+
+// Validate that containerd is running with selinux enabled in the config
```

Comment:
the doc of your function should start with the name of the func

"// TestSelinuxEnabled Validate that containerd is running with selinux enabled in the config"